config/routes.rb

```diff
@@ -178,6 +178,7 @@
 
   get '/about', to: 'pages#about', as: :about
   get '/privacy', to: 'pages#privacy', as: :privacy
+  get '/packagemanagercompatibility', to: 'pages#pmmatrix', as: :pmmatrix
```

Comment:
What about `/compatibility` instead? Can we also change the controller action name  and template name